Packs/Bonusly/Scripts/GetOwnerEmail.yml

```diff
@@ -0,0 +1,41 @@
+commonfields:
+  id: 1959e12f-4cd3-4c8d-8fd1-6c6e42a7a346
+  version: 6
+name: GetOwnerEmail
+script: |-
+  """
+  This script is used to get the owner of the incidents email
+
+  """
+
+  def get_owner_email():
+      owner_username = demisto.incidents()[0].get("owner")
```

Comment:
This will fetch the last created incident and will seek its owner. 
I am not sure this is what you wanted to achieve here could you please explain this?

Reply:
Both of these are used to get the owner of the incident's email address and utilized in the playbook as the owner in the product is only the username and not their email. For their playbook to work, you need the incident owners email.

Reply:
I see, then I believe this owner_username should be an argument for the script - that way when running the playbook you can use the incident context to fill this information and run the script.

Please remove this line and add the script argument.